src/server/api/routers/bot.ts

```diff
@@ -30,15 +31,25 @@ export const botRouter = createTRPCRouter({
         maxChars: z.number().optional(),
         maxMsgCount: z.number().optional(),
         msgCount: z.number().optional(),
-        modelId: z.nativeEnum(BotModelEnum).optional(),
+        modelId: z.nativeEnum(BotModelEnum).default(1),
```

Comment:
Let's use enum here. Not 1. 
E.g: BotModelEnum.GPT3

Reply:
So I will change 2 field use enum. 

- modelId: z.nativeEnum(BotModelEnum).default(BotModelEnum.GPT3) 
- usageLimitPerUserType: z.nativeEnum(UsageLimitTypeEnum).default(UsageLimitTypeEnum.PerDay)